CMakeLists.txt

```diff
@@ -38,6 +38,9 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 
+# Location independent code which is necessary for library
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
```

Comment:
Is this being added to targets correctly? 

This should probably be put downstream in `src/CMakeLists.txt` with the libvalhalla target, using `target_add_compile_flags_if_supported` .

Reply:
Thank you! Amended!